pkg/cmd/repo/fork/fork_test.go

```diff
@@ -687,6 +740,209 @@ func TestRepoFork_in_parent_match_protocol(t *testing.T) {
 		"Added remote.*fork")
 }
 
+func TestRepoFork_in_parent_org_survey_no(t *testing.T) {
```

Comment:
I appreciate you adding many extra tests, but all of these extra tests are completely unnecessary. The feature you've added is just unconditionally passing one extra parameter to the fork function. A single tests above is enough to exercise this functionality.